### ISSUE TITLE
Vendor asset directories: should they be supported by default?

### ISSUE THREAD

**Issue by benoitchantre** [wprig/wprig#585]
## Issue Overview

WP Rig currently doesn't support vendor assets directories: if you add a `vendor` directory in `assets/css/` or `assets/js/`, it will not be under version control nor exported by the build process when running `npm run bundle`.

Is it an enough common scenario to add support in WP Rig or should it be let to the developer?

## Possible Solution

Update `.gitignore` to include the following paths
- `!/assets/css/vendor/`
- `!/assets/js/vendor/`

Update `config.json` with:
```json
  "export": {
    "compress": true,
    "generatePotFile": true,
    "filesToCopy": [
      "LICENSE",
      "readme.txt",
      "screenshot.png",
      "assets/css/vendor/*",
      "assets/js/vendor/*"
    ]
  }
```

Update: doing that is not enough. `gulp/clean.js` (and `gulp/constants.js`) need also some changes to prevent the removal of the `vendor` directories when you run `npm run dev` and `npm run build`.

**Comment by mor10** [comment]
seems reasonable. can you make a PR?

**Comment by benoitchantre** [comment]
Yes

**Comment by ataylorme** [comment]
> Is it an enough common scenario to add support in WP Rig or should it be let to the developer?

@benoitchantre is this something you have been using? I'd like to see if other community members/WP Rig users think it would be useful as well before adding it.

**Comment by benoitchantre** [comment]
@ataylorme Yes, I have used that in a project to have a mobile menu working with [mmenu-light](https://mmenujs.com/mmenu-light/). I used their dist files.

**Comment by benoitchantre** [comment]
Closed by #587.
